content/components/selectpanel.mdx

```diff
@@ -9,51 +9,155 @@ import {Box} from '@primer/react'
-  src="https://user-images.githubusercontent.com/40274682/229011161-abc2d548-fcf1-4bed-b5a2-eb88d2121a09.png"
+  src="https://github.com/primer/design/assets/980622/17c26b4f-e4ac-4120-8714-1bfffc2b73da"
 />
-<Caption>Select panel can have as much as the branch selector component, or as little as just a multi-selection list. If the contents request user input beyond a semantic menu(example: the context menu on comments in GitHub), it’s likely you need a select panel.</Caption>
 
-### Additional control slot
+### Trigger
+
+<img width="960" alt="" src="https://github.com/primer/design/assets/980622/f6a1e670-2682-4ce7-8642-24943084c812" />
+
+Select panels can be activated by either a regular [button](/components/button), [icon button](/components/icon-button) or [action menu](/components/action-menu). If the button is intended to represent the current selection, it's crucial to have a label associated with it, either internally (within the button) or externally (adjacent to the button).
+
+### List (single select)
 
 <img
   width="960"
-  alt="Three cropped select panel footers each displaying a different option of a control in the bottom right left corner: Link, button, or checkbox input."
-  src="https://user-images.githubusercontent.com/40274682/229011396-2d2c6132-937b-49dd-974c-1a8d4d0d4737.png"
+  alt="Multi-select"
+  src="https://github.com/primer/design/assets/980622/3d0cc2b5-b958-4b62-805e-f77a9d7bf0d9"
 />
-<Caption>An optional control slot is available in the select panel’s bottom left corner. Avoid use if not absolutely necessary in order to avoid component and decision overwhelm.</Caption>
 
-### Anchoring
+If the select panel doesn't have an additional checkbox in the footer, you can make an instant selection. This means that the select panel will close as soon as you make the selection.
 
-Select panel should be anchored to its trigger. 
+However, if there is an additional form element required for that selection, such as checking an additional option, it is necessary to have a cancel and confirmation button. This prevents the dialog from unexpectedly closing when you click an option, allowing you to review and confirm your selection.
 
-<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
-  <div>
-    <img
-      src="https://user-images.githubusercontent.com/40274682/229016799-9559a4ee-0daf-42d1-b46d-11c64cf182e4.png"
-      alt=""
-    />
-    <Caption>Anchored select panel to a trigger.</Caption>
-  </div>
-</Box>
+To indicate an instant selection, we use checkmarks to show the selected option. For regular single selects, we use visually styled radio buttons. Note they are not semantically radio buttons.
```

Comment:
I'm not sure users will pick up on this distinction. If that's correct, we don't gain much by introducing a new selection indicator and it would be better to just have 1.

I don't feel strongly against this, so I'm curious what @maximedegreve and other reviewers think.

Reply:
I think I would prefer to just only use radio and checkbox styles in SelectPanel but this is not a strongly held opinion.

Reply:
Could you both go here: https://comfy-lamington-a4610e.netlify.app/explorer

And change selection to `single`.
Then toggle declarative. 

Hopefully this demonstrates a bit better the reasoning about this.

If you don't have declarative buttons the dialog becomes more a menu and the checkmarks can help indicate that. Clicking at this point this will close the dialog and instantly apply the selection.

The radio buttons on the other hand provide a more form feeling due to the declarative buttons. This design already appears in many dialog designs and feels more consistent.

Let me know what you think?